worker/src/RTC/Codecs/H264_SVC.cpp

```diff
@@ -0,0 +1,430 @@
+#define MS_CLASS "RTC::Codecs::H264_SVC"
+
+#include "RTC/Codecs/H264_SVC.hpp"
+#include "Logger.hpp"
+#include "Utils.hpp"
+
+namespace RTC
+{
+	namespace Codecs
+	{
+		/* Class methods. */
+
+		H264_SVC::PayloadDescriptor* H264_SVC::Parse(
+		  const uint8_t* data, size_t len, RTC::RtpPacket::FrameMarking* frameMarking, uint8_t frameMarkingLen)
+		{
+			MS_TRACE();
+
+			if (len < 2)
+				return nullptr;
+
+			std::unique_ptr<PayloadDescriptor> payloadDescriptor(new PayloadDescriptor());
+
+			// TODO: Pratima
```

Comment:
What is this TODO?

Reply:
Sorry that was a handled TODO. I have removed it in the new commit.